util/webhook/webhook.go

```diff
@@ -90,12 +110,37 @@ func affectedRevisionInfo(payloadIf interface{}) (string, string, bool) {
 		// Not actually sure how to check if the incoming change affected HEAD just by examining the
 		// payload alone. To be safe, we just return true and let the controller check for himself.
 		touchedHead = true
+	case bitbucketserver.RepositoryReferenceChangedPayload:
+
+		// Webhook module does not parse the inner links
+		for _, l := range payload.Repository.Links["clone"].([]interface{}) {
+			link := l.(map[string]interface{})
+			if link["name"] == "http" {
+				webURL = link["href"].(string)
+				break
+			}
+		}
+
+		// TODO: bitbucket includes multiple changes as part of a single event.
```

Comment:
TODO?

Reply:
This TODO annotation was already here for Bitbucket. Bitbucket Server use the same kind of `changes` list. I kept as it was for now.